### PR TITLE
Introduce minor improvements

### PR DESCRIPTION
**Pull request by korotkevics**
* Add a Sonar related comment
* Add GitHub Community docs
* Fix a JDeps report issue

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+### New Feature Submissions:
+
+1. [ ] Does your submission pass tests?
+2. [ ] Have you lint your code locally prior to submission?
+
+### Changes to Core Features:
+
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you written new tests for your core changes, as applicable?
+* [ ] Have you successfully ran tests with your changes locally?
```
Comment (makoJohn) [inline PULL_REQUEST_TEMPLATE.md]: Have you successfully run

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* Learn about our branching strategy: [Workflow](https://github.com/korotkevics/ships/wiki/Workflow)
+* Get a general overview of our branches: [Branches](https://github.com/korotkevics/ships/wiki/Branches)
+* Check out some Git hooks highly recommended to be used by our contributors: [Hooks](https://github.com/korotkevics/ships/wiki/Git-hooks)
+* Have a look on our On-line site and learn how to update it: [Site](https://github.com/korotkevics/ships/wiki/Online-Site)
+
+## Quick Facts
+
+* We mostly use **Waffle.io** to manage our issues and milestones, it allows us to share our current workload among ourselves and have an overview of all issues we are aware of
+* We use a private Slack channel to discuss on-going project related issues, and the channel is also integrated with **Waffle.io**
+* Insufficient test coverage ratio and/or inproper Maven/Java version will fail the build
```
Comment (makoJohn) [inline CONTRIBUTING.md]: improper

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
```
Comment (makoJohn) [inline CODE_OF_CONDUCT.md]: are not aligned with this Code of Conduct

File: CONTRIBUTING.md (hunk 2 above)
Comment (korotkevics) [reply CONTRIBUTING.md]: thanks:) I'll fix that

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (korotkevics) [reply PULL_REQUEST_TEMPLATE.md]: thanks:) I'll fix that

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (korotkevics) [reply CODE_OF_CONDUCT.md]: thanks:) I'll fix that. you've actually read it! it is generated automatically :)

Comment (makoJohn) [reply CODE_OF_CONDUCT.md]: you should put into a description of PR that this text is generated automatically

**Pull request merged**